### ISSUE TITLE
CVE-2020-24372, ingress-nginx is using the old version of Luajit, which is provided by openresty

### ISSUE THREAD

**Issue by Cyn0sure** [openresty/openresty#656]
Hi Openresty team,
ingress-nginx used Luajit 2.1, which has the issue, CVE-2020-24372. 

export LUAJIT_VERSION=31116c4d25c4283a52b2d87fed50101cf20f5b77
"https://github.com/openresty/luajit2/archive/$LUAJIT_VERSION.tar.gz"

Could you please upgrade the version of LUAJIT, 2.1.0-2020-08-27, which has a fix for CVE-2020-24372?

Thank you very much.

**Comment by agentzh** [comment]
@Cyn0sure Yeah, we'll take care of it in the next release very soon. Thanks.

**Comment by astromechza** [comment]
We're also looking forward to this fix too. Pretty difficult hand crafting a patch of a patch ourselves. 👍

**Comment by agentzh** [comment]
We just tagged v2.1-20201001 in our openresty/luajit2 repo which should include all the upstream patches:

https://github.com/openresty/luajit2/releases

Please let us know if it works for you.

Thanks!

**Comment by astromechza** [comment]
Will this be added to the openresty packaging in the next few days?

**Comment by agentzh** [comment]
@astromechza We'll definitely include this version in the next OpenResty version ASAP. But we're still doing extensive testing for the new OpenResty version in our [EC2 test cluster](https://qa.openresty.org) and process [new test regressions](https://github.com/openresty/luajit2/issues/106) of this new LuaJIT version on ARMv7. It may take longer than "a few days".

**Comment by astromechza** [comment]
Ok :) understood
